packages/marko-virtual/tests/index.test.ts

```
@@ -0,0 +1,375 @@
+ * Unit tests for @tanstack/marko-virtual
+ *
+ * These tests cover the TypeScript adapter layer — re-exports and Virtualizer
+ * construction with the options that the <virtualizer> and <window-virtualizer>
+ * tags wire up internally.
+ *
+ * Tag-level integration tests (testing the compiled Marko components with DOM
+ * mounting and reactive updates) require @marko/testing-library and a full
+ * Marko compilation pipeline. Those are not set up here but should be added
+ * as e2e tests using @marko/run's test utilities in a future iteration.
+ */
+import { describe, test, expect, beforeEach, vi } from 'vitest'
```

ESLint

[error] Member 'expect' of the import declaration should be sorted alphabetically.

(sort-imports)